Cubical/Data/Group/Base.agda

```diff
@@ -0,0 +1,22 @@
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Data.Group.Base where
+
+open import Cubical.Core.Prelude
+
+record isGroup {ℓ} (A : Set ℓ) : Set ℓ where
+  constructor group-struct
+  field
+    e  : A
+    _⁻¹  : A → A
+    _⊙_   : A → A → A
+    lUnit : ∀ a → e ⊙ a ≡ a
+    assoc  : ∀ a b c → (a ⊙ b) ⊙ c ≡ a ⊙ (b ⊙ c)
+    lCancel  : ∀ a → (a ⁻¹ ) ⊙ a ≡ e
```

Comment:
Please add the right laws to the record and a smart constructor showing that you only need one of them.  Also, please put this in a new folder called `Cubical.Algebra`.